js/src/lib/interfaces/Libraries.ts

```diff
@@ -217,13 +217,24 @@ const timm = (model: ModelData) =>
 
 model = timm.create_model("hf_hub:${model.id}", pretrained=True)`;
 
-const sklearn = (model: ModelData) =>
+const sklearn = (model: ModelData) => {
+	if (model.tags?.includes("skops")) {
+		const skopsmodelFile = model.config?.sklearn?.model?.file;
```

Comment:
Does `model.config` contain this information already? Afaik you need to add it in `moon-landing`

I did
```
from huggingface_hub import model_info
model_info("skops-ci/test-5973c788-0f03-466e-b3fd-9210443c70e3")
```

and the config is empty.


You can open a PR against https://github.com/huggingface/moon-landing/blob/main/server/repo_types/Models.ts#L353 adding a logic to add the info from skops that is of interest (the file name).

If possible it would be super nice if you could build this (we have codespaces now) to see if it works and how it looks like